src/Model/Style/InteractsWithBgColor.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpTui\Tui\Model\Style;
+
+use PhpTui\Tui\Model\AnsiColor;
+use PhpTui\Tui\Model\Color;
+use PhpTui\Tui\Model\Style;
+
+trait InteractsWithBgColor
```

Comment:
is there a reason to have 3 traits? either something can have a style or it cannot? I would call it `StyleableTrait`.

Reply:
The idea was to have more focused, smaller traits. A `StyleableTrait` with all methods looks good to me either. No objections. =D

Reply:
let's have `StyleableTrait` and if we need to break it down later we can. note that there is also just one test called `StylebaleTest` so it makes sense from that POV :slightly_smiling_face:

Reply:
I agree!